addons/html_editor/static/tests/test_helpers/user_actions.js

```diff
@@ -58,11 +58,11 @@ export async function deleteBackward(editor, isMobileTest = false) {
 
 // history
 export function undo(editor) {
-    throw new Error("Not implemented command to replace historyUndo");
+    editor.dispatch("HISTORY_UNDO");
```

Comment:
Actually this gives an extra argument for dispatching `ADD_STEP`. Currently we're setting the history by reference to the plugin in the `shared` but then using it with events. It's inconsistent.